### PR TITLE
Ecom Homepage Design 

### PR DESCRIPTION
**Pull request by rishichourasia**
image
image

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (AnuragSahu11) [inline .vscode/settings.json:1]: remove this file or git ignore it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		<header class="nav fixed">
+			<div class="menu-logo">
+				<div class="menu-btn">
+					<img
+						class="logo-svg"
+						src="./assets/logo/logo simple dark.svg"
+						alt="logo-svg"
+					/>
+				</div>
+				<!-- <div class="logo">
```
Comment (AnuragSahu11) [inline index.html:34]: remove unwanted comment

File: stylesheets/header.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.nav {
+	margin-top: 0;
+	background-color: rgb(245, 245, 245);
+	color: black;
```
Comment (AnuragSahu11) [inline stylesheets/header.css:4]: use variable

File: stylesheets/index.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.image-text-overlay .img-text {
+	font-family: "PT Sans Narrow", sans-serif;
+	font-size: 3rem;
+	text-align: center;
+	color: white;
```
Comment (AnuragSahu11) [inline stylesheets/index.css:27]: use variable

**Review by AnuragSahu11 — APPROVED** [review]
Good work , you can use variables for few things, other than that you can merge

File: stylesheets/index.css (hunk 4 above)
Comment (rishichourasia) [reply stylesheets/index.css:27]: alright , got it , will do the necassary changes in the next commit

**Pull request merged**